packages/java/gradle-plugin/src/functionalTest/kotlin/com/vaadin/hilla/gradle/plugin/AbstractGradleTest.kt

```diff
@@ -25,7 +25,7 @@ import org.junit.Before
  */
 abstract class AbstractGradleTest {
 
-    val hillaVersion = System.getenv("hilla.version").takeUnless { it.isNullOrEmpty() } ?: "24.6-SNAPSHOT"
+    val hillaVersion = System.getenv("hilla.version").takeUnless { it.isNullOrEmpty() } ?: "24.7-SNAPSHOT"
```

Comment:
Is there a reason that flow shouldn't be updated to `24.7-SNAPSHOT`?

Reply:
No, but it is not related to this PR. It has some issues as you can see in #2961